code/modules/mob/living/carbon/human/species/species.dm

```diff
@@ -461,7 +461,13 @@
 		H.see_in_dark = 8
 		if(!H.druggy)
 			H.see_invisible = SEE_INVISIBLE_LEVEL_TWO
+		if(H.client)
+			H.overlay_fullscreen("pmaster_lighting_g", /obj/screen/fullscreen/pmaster_g_default, 1)
```

Comment:
you call this exact proc three times, when overlay_fullscreen should only be called when shit actually needs to be updated